remotespark/utils/eventshandler.py

```diff
@@ -0,0 +1,13 @@
+from remotespark.utils.log import Log
+
+
+class EventsHandler:
+    def __init__(self):
+        self.logger = Log("EventsHandler")
+
+    def store_event(self, *args):
+        """
+        Storing the Event details using the logger.
+        """
+        event_line = ",".join("{}".format(arg) for arg in args)
```

Comment:
If we want to unify the API across the different handlers the list will be more fit. on the other side, key-value will give more sense to the logs information.  So, I go for the current state, what do you think?